
freebsd-buildutils 8.2 doesn't support MK_INET_SUPPORT nor MK_OFED.

--- a/sys/modules/Makefile
+++ b/sys/modules/Makefile
@@ -361,12 +361,11 @@
 .endif
 .endif
 
-.if ${MK_INET_SUPPORT} != "no" || defined(ALL_MODULES)
+.if 1
 _if_gre=	if_gre
 .endif
 
-.if (${MK_INET_SUPPORT} != "no" || ${MK_INET6_SUPPORT} != "no") || \
-	defined(ALL_MODULES)
+.if 1
 _if_carp=	if_carp
 .endif
 
@@ -382,8 +381,7 @@
 _netgraph=	netgraph
 .endif
 
-.if (${MK_PF} != "no" && (${MK_INET_SUPPORT} != "no" || \
-	${MK_INET6_SUPPORT} != "no")) || defined(ALL_MODULES)
+.if 1
 _pf=		pf
 _pflog=		pflog
 .if ${MK_INET_SUPPORT} != "no"
@@ -442,7 +440,7 @@
 _linsysfs=	linsysfs
 _linux=		linux
 _mse=		mse
-.if ${MK_OFED} != "no" || defined(ALL_MODULES)
+.if 1
 _mlx4=		mlx4
 _mlx4ib=	mlx4ib
 _mlxen=		mlxen
@@ -594,7 +592,7 @@
 _linsysfs=	linsysfs
 _linux=		linux
 _mly=		mly
-.if ${MK_OFED} != "no" || defined(ALL_MODULES)
+.if 1
 _mlx4=		mlx4
 _mlx4ib=	mlx4ib
 _mlxen=		mlxen
